src/component/Tooltip.tsx

```diff
@@ -99,6 +100,16 @@ export type TooltipProps<TValue extends ValueType, TName extends NameType> = Omi
   trigger?: TooltipTrigger;
   useTranslate3d?: boolean;
   wrapperStyle?: CSSProperties;
+  /**
+   * Tooltip always attaches itself to "categorical" axis. Which axis is categorical? Depends on the layout:
```

Comment:
What about when both domains are numeric? It still treats one as categorical?

Reply:
Yeah the type has nothing to do with it. Unfortunate naming.

https://github.com/recharts/recharts/blob/849d7b04faac8848cfbaf59957e861ed01ef4059/src/util/ChartUtils.ts#L295

Reply:
Lol

Reply:
Also see https://github.com/recharts/recharts/blob/849d7b04faac8848cfbaf59957e861ed01ef4059/src/chart/generateCategoricalChart.tsx#L578

Reply:
I want to get rid of this completely later but for now I will update the comment.

Reply:
Sounds good